cmake/config.cmake.in

```diff
@@ -0,0 +1,40 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+# Add FindRdKafka.cmake
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}")
+
+# Find boost optional
+find_dependency(Boost REQUIRED)
+
+# Try to find the RdKafka configuration file if present.
+# This will search default system locations as well as RdKafka_ROOT and RdKafka_Dir paths if specified.
+find_dependency(RdKafka QUIET CONFIG)
+set(RDKAFKA_TARGET_IMPORTS ${RdKafka_FOUND})
+if (NOT RdKafka_FOUND)
+    message(STATUS "RdKafkaConfig.cmake not found. Please set RDKAFKA_ROOT or RDKAFKA_DIR if a config file is installed. Attempting to find module instead...")
```

Comment:
Following up on the comment in the top level `CMakeLists.txt` file, this shouldn't be here either unless there's an error.

Reply:
OK...see my previous comment. Since the find_package searches config and module, it's good for debugging to know what's going on. Maybe I can put this under CMAKE_VERBOSE_MAKEFILE or something like that?

Reply:
If you want it for debugging purposes, this probably shouldn't be in this file but in the main CMake file instead (under some flag). A user importing cppkafka on their project (via this script) shouldn't care at all about how rdkafka is being found: they just care about the `cppkafka` target either being there or not after running the script.

Reply:
Ok I can leave it outside the CppKafkaConfig.cmake and only include it in the top CMakeLists of the CppKafka project itself. In that case you do want to know why you can't find RdKafka.

Reply:
BTW the REQUIRE makes sure that an error message will be printed if the module is not found.